host-services/builtins/keyvalue.go

```diff
@@ -15,17 +14,15 @@ const kvServiceMethodGet = "get"
 const kvServiceMethodSet = "set"
 const kvServiceMethodDelete = "delete"
 const kvServiceMethodKeys = "keys"
-const defaultKVBucketName = "hs_%s_kv"
 
 type KeyValueService struct {
 	log    *slog.Logger
 	config kvConfig
 }
 
 type kvConfig struct {
-	BucketName   string `json:"bucket_name"`
-	MaxBytes     int    `json:"max_bytes"`
-	JitProvision bool   `json:"jit_provision"`
```

Comment:
Now that I see this implemented, I actually think it would have been good to keep JIT provisioning of resources.